db_anonymization.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+abs_path='/path/to/expertiza/on/server'
+
+cd $abs_path
+
+gem install bundler -v 1.16.6
```

Comment:
This command only needs to be run once on the system that is being used for anonymization